src/Interfaces/SelfHandlingCommandInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace davidjeddy\StripeCB\Interfaces;
+
+use davidjeddy\StripeCB\Interfaces\HandlerInterface;
+
+/**
+ * Interface SelfHandlingCommandInterface
+ *
+ * @author Davidj J Eddy <me@davidjeddy.com>
+ *
+ */
+interface SelfHandlingCommandInterface extends HandlerInterface
+{
+
+}
```

Comment:
Empty interface?

Reply:
Hold over from source implementation. Keeping for future iteration.

Reply:
I'm generally not a fan of this. While I understand why it's happening, this is quick way to let dead code hang around.

Reply:
Indeed, solid point. Removed.